backend/src/csv_handler/central_config/env.py

```diff
@@ -13,3 +13,4 @@
 USERPOOL_ID = getenv("COGNITO_USERPOOL_ID", "COGNITO_USERPOOL_ID is not set")
 APP_CLIENT_ID = getenv("COGNITO_APP_CLIENT_ID", "APP_CLIENT_ID is not set")
 OTC_CLIENT_API_URL = getenv("OTC_CLIENT_API_URL", "OTC_CLIENT_API_URL is not set")
+BUCKET_NAME = getenv("S3_BUCKET_NAME")
```

Comment:
```suggestion
BUCKET_NAME = getenv("CLAMAV_S3_BUCKET_NAME")
```
More descriptive - this bucket will only contain scan artefacts